Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ObjC/include/FBSnapshotTestCase_Accessibility.h

```diff
@@ -14,7 +14,11 @@
 //  limitations under the License.
 //
 
-#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+//#if SWIFT_PACKAGE
+@import iOSSnapshotTestCaseCore;
+//#else
+//#import <iOSSnapshotTestCaseCore/FBSnapshotTestCase.h>
+//#endif
```

Comment:
This is the problem. Ideally this should be
```c
#if SWIFT_PACKAGE
@import iOSSnapshotTestCaseCore;
#else
#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
#endif
```
(I think the `@import` should actually be `#import` with a specific file, but that wouldn't compile 🤷‍♂️)

The issue is that `SWIFT_PACKAGE` isn't available in public header files as far as I can tell, so there's no way to distinguish whether it's being built with SPM or CP.

@alanzeino any idea how to work around this?

Reply:
Nevermind, turns out this import isn't actually needed at all 🤦‍♂️